helper/token.js

```diff
@@ -0,0 +1,79 @@
+const promise = require('bluebird');
+const dbHelper = require('../helper/database');
+const request = require('request');
+const tokenCollection = 'token';
+const canvasUrl = 'http://canvas.differ.chat/api/v1/users/36/courses?access_token=';
```

Comment:
In #8 the id is replaced with self, but not moved the url in config, good catch. I will move the canvasUrl under config/settings in #8